operator/src/network/controller.rs

```diff
@@ -2974,4 +3003,30 @@ mod tests {
             .expect("reconciler");
         timeout_after_1s(mocksrv).await;
     }
+    #[tokio::test]
+    #[traced_test]
+    async fn reconcile_expired() {
+        // Expect no calls
+        let mock_rpc_client = MockIpfsRpcClientTest::new();
+
+        let (testctx, api_handle) = Context::test(mock_rpc_client);
+        let fakeserver = ApiServerVerifier::new(api_handle);
+        let mut network = Network::test().with_spec(NetworkSpec {
+            // Set expiration as now
+            ttl_seconds: Some(0),
+            ..Default::default()
+        });
+        // Set creation time as now.
```

Comment:
can we add a test where ttl and creation time aren't the same so we make sure the arithmetic order/sign is correct?

Reply:
done, added both an expire and not expired test